### PR TITLE
40K 11 + 3 tokens + 1 cleanup

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/drach_nyen.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Drach'Nyen
+ManaCost:4 B R
+Types:Legendary Artifact Equipment
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigExile | TriggerDescription$ Echo of the First Murder — When CARDNAME enters the battlefield, exile up to one target creature.
+SVar:TrigExile:DB$ ChangeZone | Imprint$ True$ | Origin$ Battlefield | Destination$ Exile | ValidTgts$ Creature
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/drach_nyen.txt:5]: "up to" not handled

File: forge-gui/res/cardsfolder/upcoming/epistolary_librarian.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Epistolary Librarian
+ManaCost:2 W U
+Types:Creature Astartes Wizard
+PT:3/4
+T:Mode$ Attacks | ValidCard$ Card.Self  | Execute$ TrigCast | TriggerDescription$ Veil of Time — Whenever CARDNAME attacks, you may cast a spell with mana value X or less from your hand without paying its mana cost, where X is the number of attacking creatures.
+SVar:TrigCast:DB$ Play | ValidZone$ Hand | Valid$ Card.cmcLEX | ValidSA$ Spell | Optional$ True | WithoutManaCost$ True |
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/epistolary_librarian.txt:6]: `+YouOwn`

File: forge-gui/res/cardsfolder/upcoming/epistolary_librarian.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Name:Epistolary Librarian
+ManaCost:2 W U
+Types:Creature Astartes Wizard
+PT:3/4
+T:Mode$ Attacks | ValidCard$ Card.Self  | Execute$ TrigCast | TriggerDescription$ Veil of Time — Whenever CARDNAME attacks, you may cast a spell with mana value X or less from your hand without paying its mana cost, where X is the number of attacking creatures.
+SVar:TrigCast:DB$ Play | ValidZone$ Hand | Valid$ Card.cmcLEX | ValidSA$ Spell | Optional$ True | WithoutManaCost$ True | 
+SVar:Z:Count$Valid Creature.attacking+YouCtrl
+SVar:HasAttackingEffect:TRUE
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/epistolary_librarian.txt:8]: `HasAttackEffect`

File: forge-gui/res/cardsfolder/upcoming/inquisitor_eisenhorn.txt [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Name:Inquisitor Eisenhorn
+ManaCost:2 U B
+Types:Legendary Creature Human Inquisitor
+PT:2/3
+T:Mode$ Drawn | ValidCard$ Card.YouOwn | Number$ 1 | OptionalDecider$ You | Static$ True | ForReveal$ True | Execute$ DBReveal | TriggerZones$ Battlefield | TriggerDescription$ You may reveal the first card you draw each turn as you draw it. Whenever you reveal an instant or sorcery card this way, create Cherubael, a legendary 4/4 black Demon creature token with flying.
+SVar:DBReveal:DB$ Reveal | Defined$ You | RevealDefined$ TriggeredCard | RememberRevealed$ True | SubAbility$ DBToken
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/inquisitor_eisenhorn.txt:6]: No reason to template this differently than God-Eternal Kefnet by leaving out the real trigger there

File: forge-gui/res/cardsfolder/upcoming/inquisitor_eisenhorn.txt [hunk 5]
```diff
@@ -3,8 +3,9 @@ ManaCost:2 U B
 Types:Legendary Creature Human Inquisitor
 PT:2/3
 T:Mode$ Drawn | ValidCard$ Card.YouOwn | Number$ 1 | OptionalDecider$ You | Static$ True | ForReveal$ True | Execute$ DBReveal | TriggerZones$ Battlefield | TriggerDescription$ You may reveal the first card you draw each turn as you draw it. Whenever you reveal an instant or sorcery card this way, create Cherubael, a legendary 4/4 black Demon creature token with flying.
-SVar:DBReveal:DB$ Reveal | Defined$ You | RevealDefined$ TriggeredCard | RememberRevealed$ True | SubAbility$ DBToken
-SVar:DBToken:DB$ Token | TokenOwner$ You |  SubAbility$ DBCleanup | TokenScript$ cherubael | ConditionDefined$ Remembered | ConditionPresent$ Instant,Sorcery
+SVar:DBReveal:DB$ Reveal | Defined$ You | RevealDefined$ TriggeredCard | RememberRevealed$ True | SubAbility$ DBTrigger
+SVar:DBTrigger:DB$ ImmediateTrigger | RememberObjects$ RememberedCard | ConditionDefined$ Remembered | ConditionPresent$ Instant,Sorcery | SubAbility$ DBCleanup | Execute$ DBToken | TriggerDescription$ Whenever you reveal an instant or sorcery card this way, create Cherubael, a legendary 4/4 black Demon creature token with flying.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/inquisitor_eisenhorn.txt:7]: No need for RememberObjects

File: forge-gui/res/cardsfolder/upcoming/commissar_severina_raine.txt [hunk 6]
```diff
@@ -0,0 +1,12 @@
+Name:Commissar Severina Raine
+ManaCost:1 W B
+Types:Legendary Creature Human Soldier
+PT:2/2
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigLoseLife | TriggerDescription$ Leading from the Front — Whenever CARDNAME attacks, each opponent loses X life, where X is the number of other attacking creatures.
+SVar:TrigLoseLife:DB$ LoseLife | Defined$ Opponent | LifeAmount$ X 
+A:AB$ Draw | Cost$ T Sac<1/Creature.Other/another creature> | PrecostDesc$ Summary Execution — | SubAbility$ DBGainLife | SpellDescription$ You gain 2 life and draw a card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/commissar_severina_raine.txt:7]: Effect order is wrong

File: forge-gui/res/cardsfolder/upcoming/company_commander.txt [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+Types:Creature Human Soldier
+PT:2/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ Command Section — When CARDNAME enters the battlefield, create a number of 1/1 white Soldier creature tokens equal to the number of opponents you have.
+SVar:TrigToken:DB$ Token | TokenAmount$ X | TokenScript$ w_1_1_soldier | TokenOwner$ You
+SVar:X:PlayerCountOpponents$Amount
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigPump | NoResolvingCheck$ True | TriggerDescription$ Bring it Down! — Whenever CARDNAME attacks, creatures you control gain deathtouch until end of turn.
+SVar:TrigPump:DB$ PumpAll | ValidCards$ Creature.YouCtrl | KW$ Deathtouch
+SVar:HasAttackEffect:TRUE
+DeckHas:Keyword$Deathtouch & Ability$Token
+Oracle:Command Section — When Company Commander enters the battlefield, create a number of 1/1 white Soldier creature tokens equal to the number of opponents you have.\NBring it Down! — Whenever Company Commander attacks, creatures you control gain deathtouch until end of turn.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/company_commander.txt:12]: \N

File: forge-gui/res/tokenscripts/c_4_4_a_robot_noblock.txt [hunk 8]
```diff
@@ -0,0 +1,6 @@
+Name:Robot Token
+ManaCost:no cost
+Types:Artifact Creature Robot
+PT:4/4
+K:CARDNAME can't block.
```
Comment (Northmoc) [inline forge-gui/res/tokenscripts/c_4_4_a_robot_noblock.txt:5]: do this like #1551 
https://github.com/Card-Forge/forge/pull/1551/files#diff-694431d534d1d47f75f2289bb6c5bb6b563284c2be8247b07c9b44eab64992ef

File: forge-gui/res/tokenscripts/c_4_4_a_robot_noblock.txt [hunk 9]
```diff
@@ -0,0 +1,6 @@
...
+Oracle:This creature can't block
```
Comment (Northmoc) [inline forge-gui/res/tokenscripts/c_4_4_a_robot_noblock.txt:6]: needs `.`

File: forge-gui/res/cardsfolder/upcoming/inquisitor_eisenhorn.txt [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+SVar:DBReveal:DB$ Reveal | Defined$ You | RevealDefined$ TriggeredCard | RememberRevealed$ True | SubAbility$ DBTrigger
+SVar:DBTrigger:DB$ ImmediateTrigger | ConditionDefined$ Remembered | ConditionPresent$ Instant,Sorcery | SubAbility$ DBCleanup | Execute$ DBToken | TriggerDescription$ Whenever you reveal an instant or sorcery card this way, create Cherubael, a legendary 4/4 black Demon creature token with flying.
+SVar:DBToken:DB$ Token | TokenScript$ cherubael
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | Execute$ TrigInvestigate | CombatDamage$ True | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, investigate that many times. (Create a Clue token. It's an artifact with "{2}, Sacrifice this artifact: Draw a card.")
+SVar:TrigInvestigate:DB$ Investigate | Num$ X | Defined$ You
+SVar:X:TriggerCount$DamageAmount
+DeckHas:Type$Clue|Demon & Ability$Token
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/inquisitor_eisenhorn.txt:13]: add Artifact to Type and Sacrifice to Ability

**Comment by Agetian** [comment]
Looks good to me :+1:

File: forge-gui/res/cardsfolder/upcoming/drach_nyen.txt [hunk 11]
```diff
@@ -0,0 +1,10 @@
+Name:Drach'Nyen
+ManaCost:4 B R
+Types:Legendary Artifact Equipment
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigExile | TriggerDescription$ Echo of the First Murder — When CARDNAME enters the battlefield, exile up to one target creature.
+SVar:TrigExile:DB$ ChangeZone | Imprint$ True$ | TargetMin$ 0 | TargetMax$ 1 | Origin$ Battlefield | Destination$ Exile | ValidTgts$ Creature
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ X | AddKeyword$ Menace | Description$ Daemon Sword — Equipped creature has menace and gets +X/+0, where X is the exiled card's power.
+K:Equip:2
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/drach_nyen.txt:7]:
```
T:Mode$ ChangesZone | Origin$ Battlefield | ValidCard$ Card.Self | Destination$ Any | Execute$ DBCleanup | Static$ True
T:Mode$ ChangesZone | Origin$ Exile | Destination$ Any | Static$ True | ValidCard$ Card.IsImprinted+ExiledWithSource | Execute$ DBForget
SVar:DBForget:DB$ Pump | ForgetImprinted$ TriggeredCard
SVar:DBCleanup:DB$ Cleanup | ClearImprinted$ True
```

Also
`SVar:NeedsToPlay:Creature.OppCtrl`
imo

**Review by tool4ever — APPROVED** [review]

**Pull request merged**